app/src/main/java/org/fossasia/susi/ai/skills/aboutus/AboutUsFragment.kt

```diff
@@ -43,29 +45,37 @@ class AboutUsFragment : Fragment() {
 
     @NonNull
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-        about_susi.setOnClickListener({
+
+        about_susi.text = Html.fromHtml(getString(R.string.susi_about))
+        contributors_desc.text = Html.fromHtml(getString(R.string.susi_contributors_desc))
+        susi_skill_cms_desc.text = Html.fromHtml(getString(R.string.susi_skill_cms_desc))
+        susi_license_info_desc.text = Html.fromHtml(getString(R.string.susi_license_information_desc))
+        susi_report_issues_desc.text = Html.fromHtml(getString(R.string.susi_report_issues_desc))
```

Comment:
This is deprecated, just create a method which checks API and returns correct HTML call

Reply:
@iamareebjamal Added method. Please review.